test/integration/merkle-lockup/ll/clawback/clawback.t.sol

```diff
@@ -23,27 +26,57 @@ contract Clawback_Integration_Test is MerkleLockup_Integration_Test {
         _;
     }
 
-    function test_RevertGiven_CampaignNotExpired() external whenCallerAdmin {
+    function test_Clawback_BeforeFirstClaim() external whenCallerAdmin {
+        test_Clawback(users.admin);
+    }
+
+    modifier AfterFirstClaim() {
```

Comment:
the modifier name does not use camelCase: `afterFirstClaim`

Reply:
Oh right. Sorry.